src/views/error-not-found.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "layout.njk" %}
+
+{% from 'govuk/components/details/macro.njk' import govukDetails %}
+
+{% block pageTitle %}
+  Appeal a late filing penalty
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        Page not found
+      </h1>
+      <p>If you typed the web address, check it is spelled correctly.</p>
+      <p>If you pasted the web address, check you copied the entire address.</p>
+      <p>If the web address is correct or you selected a link, the page may have moved.
+        <a href="/appeal-a-penalty">Go to the Appeal a late filing penalty start page</a>.
+
+      <div class="govuk-form-group">
+        {% set otherWaysToAppeal %}
+          <p>You can send an appeal to us by email or by post. It should include a full explanation of your appeal.</p>
+          <h2 class="govuk-heading-m">By email</h2>
+          <p class="govuk-body">{{ enquiryEmail }}</p>
```

Comment:
strangely when i run this locally i see the email address displayed in single quotes, can you double check this isn't happening for you?

Reply:
Wasn't happening when running on vagrant, but can confirm it does on the docker

Reply:
if you pull the latest version of the poc branch we are using it should be fixed